var/spack/repos/builtin/packages/cantera/package.py

```diff
@@ -43,6 +45,8 @@ class Cantera(SConsPackage):
     # Matlab toolbox dependencies
     extends('matlab', when='+matlab')
 
+    conflicts('~sundials', when='@2.3.0:')
```

Comment:
Did you forget to make the dependency itself optional:
```python
depends_on('sundials', when='+sundials')
```
or am I missing something?

Reply:
Thanks, should be fixed now.